README.md

```diff
@@ -67,6 +67,8 @@ This file contains a list of non-web wasm runtimes in development in no particul
 
 30. **[m3](https://github.com/soundandform/m3)** - C
 
+31. **[wasvm](https://github.com/koagi/wasvm)** - Rust
```

Comment:
```suggestion
31. **[wasvm](https://github.com/kogai/wasvm)** - Rust
```
I guess :)

Reply:
Wow, that's my fault 🙈 
Thanks to pointing out 🙏